git-branchless-submit/src/lib.rs

```diff
@@ -422,7 +431,8 @@ fn submit(
     if !skipped_branch_names.is_empty() {
         writeln!(
             effects.get_output_stream(),
-            "Skipped {} (already up-to-date): {}",
+            "{} {} (already up-to-date): {}",
+            if dry_run { "Would skip" } else { "Skipped" },
```

Comment:
The tense of these messages felt relevant to change in this context, but another option would be to leave there messages as-is and instead display a separate message w/ something like `This is a DRY RUN; no branches have been pushed or created.`

Reply:
The tense change is nice in my opinion. If you've already done it, then there's no need to change it to the other way.